contrib/api-setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh --
```

Comment:
I can call it whatever you want. 

I pulled the script out at the last minute. Probably ought to mention it.